src/storage/mvcc/write.rs

```diff
@@ -20,6 +20,8 @@ const FLAG_DELETE: u8 = b'D';
 const FLAG_LOCK: u8 = b'L';
 const FLAG_ROLLBACK: u8 = b'R';
 
+const FLAG_PROTECTED: u8 = b'p';
```

Comment:
I don't think it is a good idea to add a such field for write.

Reply:
We can put a short value to rollback key without the flag.

Reply:
It's a compatible change.

Reply:
Version control is needed.

Reply:
We never reads the value of a Rollback key, so it's safe to put a value in it.

Reply:
Before we know it is rollback, we should call the `parse` function of Write type.

Reply:
I mean remove the `FLAG_PROTECTED` and put a value under `SHORT_VALUE_PREFIX` flag.

Reply:
For example, in `collapse_prev_rollback`, it will call seek_write to check if the previous commit is rollback and extract the ts.

Reply:
It's workable but a bit tricky. I can change the PR to @coocood 's plan. But in the end we need something like version control.

Reply:
> I mean remove the `FLAG_PROTECTED` and put a value under `SHORT_VALUE_PREFIX` flag.

You mean reuse short value flag for Rollback, emm, it works but weird. Maybe the value set as "protected" looks a little comfortable.

Reply:
Is the short value only used for compatibility consideration?

Reply:
Yes.